etc/configuration.ini.sample

```diff
@@ -38,10 +38,12 @@ SSL: True
 Certificate: ssl/cve-search.crt
 Key: ssl/cve-search.key
 WebInterface: Full
-MountPath: /MOUNTY/MC/MOUNT
```

Comment:
This additional space at the end of line is probably here by mistake as this code is written on a phone.